Src/FluentAssertions/AndWhichConstraint.cs

```diff
@@ -43,10 +43,8 @@ public AndWhichConstraint(TParentConstraint parentConstraint, IEnumerable<TMatch
                     matchedElements.Select(
                         ele => "\t" + Formatter.ToString(ele)));
 
-                string message = string.Format(
-                    "More than one object found.  FluentAssertions cannot determine which object is meant.  Found objects:{0}{1}",
-                    Environment.NewLine,
-                    foundObjects);
+                string message = "More than one object found.  FluentAssertions cannot determine which object is meant."
```

Comment:
Would have been nice if these changes were kept in a separate commit. The PR changes now mix the invariant changes and other things.

Reply:
It is related, as it otherwise triggered `string.Format` without providing an `IFormatProvider`.

I can split the commit up into two with:
* the cases that rightfully needed to ignore culture, and
* those where the changes are to please the analyzer. (e.g. when all arguments to be formatted are already strings)

Reply:
Never mind.